### ISSUE TITLE
Create New Letter Content for the Portal

### ISSUE THREAD

**Issue by edobrowolska** [ESA-EarthCODE/EarthCODE#2]
Hi @edobrowolska, 
I assigned this to you, because you normally seem to coordinate these activities.  Please let me know if I am wrong.
Thanks
Garin

**Comment by edobrowolska** [comment]
Right, here as well I think together with @aapopescu about the first content to the Newsletter, which will be for sure a summary of kick off and upcoming activities, roadmap etc. I will think of what we can add there

**Comment by GarinSmith** [comment]
As discussed yesterday, we will plan for a target release on 20th December subject to confirmation from @Schpidi

**Comment by Schpidi** [comment]
December 20th sounds reasonable assuming that required content (see https://github.com/ESA-EarthCODE/portal/issues/33) is available early enough to integrate it, ideally a week in advance.

Cc @silvester-pari

**Comment by silvester-pari** [comment]
Currently we link to https://esacontact.esa.int/ESA_EO_OpenScience_Subscribe/. If there is any other link for newsletter sign up, we can of course change that link.

**Comment by GarinSmith** [comment]
I am marking this old user story as complete. 
We now have an established solution for Newsletter content.
See - https://earthcode.esa.int/news-events
@gcooksley
